packages/core/content-releases/server/src/utils/index.ts

```diff
@@ -1,16 +1,16 @@
-import type { UID, Data } from '@strapi/types';
+import type { UID, Data, Core } from '@strapi/types';
 
 export const getService = (
   name: 'release' | 'release-validation' | 'scheduling' | 'release-action' | 'event-manager',
-  { strapi } = { strapi: global.strapi }
+  { strapi }: { strapi: Core.Strapi }
```

Comment:
this was assigned to the global strapi by default, couldn't this be an issue?

Reply:
This was not a good pattern & an ensured this is a required param & is passed everywhere now